daemon/kill.go

```diff
@@ -105,6 +115,9 @@ func (daemon *Daemon) Kill(container *container.Container) error {
 		return errNotRunning{container.ID}
 	}
 
+	// FIXME(vdemeester) should we disable restartpolicy for SIGKILL or not ?
```

Comment:
Ok after some discussion; it's better to keep the current behavior here; can your remove this?